.scripts/build_steps.sh

```diff
@@ -34,9 +34,9 @@ CONDARC
 export CONDA_LIBMAMBA_SOLVER_NO_CHANNELS_FROM_INSTALLED=1
 
 mamba install --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build boa conda-forge-ci-setup=4
+    pip mamba conda-build=3.27.0 boa conda-forge-ci-setup=4
```

Comment:
These are manual changes _after_ the rerender, right? Not a super-fan of this option since it gets reset after every rerender.
There probably is no option to set this in `conda-forge.yml`, right?

Reply:
Yes, true, no.

I think this will be fixed in the next release, so hopefully it will only be necessary for a couple of days

Reply:
Does it really fix the issue? The windows and osx builds still take very long

<img width="854" alt="image" src="https://github.com/conda-forge/rattler-build-feedstock/assets/29506042/8e87cbb8-78f3-4093-819f-2d13d9f94bd2">


`.sh` files are probably not executed on windows, it probably also needs to be adjusted in https://github.com/conda-forge/rattler-build-feedstock/blob/7fd547c5bbfd9e56ef8b77cdba695227d29d63d3/.scripts/run_win_build.bat#L27 and https://github.com/conda-forge/rattler-build-feedstock/blob/7fd547c5bbfd9e56ef8b77cdba695227d29d63d3/.scripts/run_osx_build.sh#L29-L31